qemu/tests/ceph_image_mem_leak.py

```diff
@@ -0,0 +1,48 @@
+import time
+
+from avocado.utils import process
+
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Start guest with 10 rbd data disks and system disk.
+    2) Check the block info for 2 hours
+    3) Check the used memory size is not increased
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+
+    vm = env.get_vm(params["main_vm"])
+    vm.verify_alive()
+
+    # delay 2 minutes to wait the guest boot up completely
+    time.sleep(120)
+
+    pid = vm.process.get_pid()
+    qemu_stat_file = "/proc/" + str(pid) + "/status"
+    used_mem_cmd = "cat %s | grep VmRSS | awk '{print $2}'" % qemu_stat_file
+    used_mem_size_before = process.system_output(used_mem_cmd, shell=True)
+
+    test.log.info("Begin to query blocks for 2 hours.")
+    timeout = time.time() + 60 * 60 * 2  # 2 hours from now
+    while True:
+        if time.time() > timeout:
+            break
+        vm.monitor.cmd("query-blockstats")
+        vm.monitor.cmd("query-block")
+
+    used_mem_size_after = process.system_output(used_mem_cmd, shell=True)
+    test.log.info("Check whether the used memory size is increased.")
+    if int(used_mem_size_after) > int(used_mem_size_before) * 1.2:
+        test.fail(
+            "The used memory size before is %ds, but after checking the blocks "
+            "for 2 hours, it increased to %ds. There should be memory leaks, "
+            "check please." % (
+                used_mem_size_before, used_mem_size_after))
+
+    vm.destroy()
```

Comment:
as you set kill_vm = yes, then vt can do the 'destroy' work, but this is also OK for me.